src/views/bouquets/BouquetEditView.vue

```diff
@@ -23,27 +30,111 @@ const steps = [
   'Composition du bouquet',
   'Récapitulatif'
 ]
+const selectedThematique = ref('Produire')
+const selectedSubTheme = ref('Nucléaire')
 const currentStep = ref(1)
+const urlData = ref()
+
+const subThemes = computed(() => {
+  return getOptions(selectedThematique.value)
+})
+
+const loadDatasets = async (datasetIds, bouquet) => {
+  for (const datasetId of datasetIds) {
+    const dataset = await datasetStore.load(datasetId)
+    datasets.value.push({
+      dataset,
+      description: bouquet.extras[`${config.universe_name}:${dataset.id}:description`] || "",
+    })
+  }
+}
+
+const search = async (query) => {
+  if (!query) return []
+  const results = await searchAPI._search(query, { page_size: 10 })
+  return results.data.map(r => {
+    return { value: r.id, label: r.title }
+  }).filter(r => !datasets.value.map(d => d.dataset.id).includes(r.value))
+}
+
+const onSubmitModal = async () => {
+  if (!selectedDataset.value.id) return
+  if (isEditDesc.value) {
+    const idx = datasets.value.findIndex(d => d.dataset.id === selectedDataset.value.id)
+    datasets.value[idx].description = selectedDataset.value.description
+  } else {
+    const dataset = await datasetStore.load(selectedDataset.value.id)
+    if (dataset) {
+      datasets.value.push({ dataset, description: selectedDataset.value.description })
+    }
+  }
+  selectedDataset.value = {}
+  isEditDesc.value = false
+}
+
+const onDeleteDataset = (datasetId) => {
+  datasets.value = datasets.value.filter(d => d.dataset.id !== datasetId)
+  delete loadedBouquet.value.extras[`${config.universe_name}:${datasetId}:description`]
+}
+
+const onEditDataset = (dataset) => {
+  isModalOpen.value = true
+  isEditDesc.value = true
+  selectedDataset.value.id = dataset.id
+  selectedDataset.value.title = dataset.title
+  selectedDataset.value.description = datasets.value.find(d => d.dataset.id === dataset.id).description
+}
+
+const extrasFromDatasets = () => {
+  const extras = {}
+  for (const dataset of datasets.value) {
+    extras[`${config.universe_name}:${dataset.dataset.id}:description`] = dataset.description
+  }
+  return extras
+}
+
+const validateAndMoveToStep = (newStep) => {
+  if (!form.value.name || !form.value.description) {
+    errorMessage.value = 'Merci de bien remplir les champs'
+  } else {
+    currentStep.value = newStep
+  }
+}
 
 const onSubmit = async () => {
   let res
   const data = {
     ...form.value
   }
+
+  const informationsInExtras = {
+    informations: [{
+      'theme': selectedThematique.value,
```

Comment:
`selectedThematique` -> `selectedTheme` ?
D'autant plus qu'il y a déjà `selectedSubTheme`

Reply:
I changed the name